app/views/index.html

```diff
@@ -28,19 +28,36 @@ <h1>
 
       <!-- Add your content here -->
 
-      <p class="nhsuk-body-l">This kit lets you rapidly create HTML prototypes of NHS services.</p>
+      <p class="nhsuk-body-l">
+        This kit lets you rapidly create HTML prototypes of NHS services.
+      </p>
 
-      <h2 class="nhsuk-heading-m">About this page</h2>
+      <h2 class="nhsuk-heading-m">
+        About this page
+      </h2>
 
-      <p>Use this page as the index for your project.</p>
+      <p>
+        Use this page as the index for your project.
+      </p>
       
-      <p>You'll find this page in the kit at <strong>/app/views/index.html</strong>.</p>
+      <p>
+        The code for this page in the kit, you can find it at <strong>/app/views/index.html</strong>.
```

Comment:
For consistency... when telling users to look in a folder, we've used '/' before the folder here, however on other examples such as on the `docs/views/how-tos/passing-data-page.html` page. There is no '/' before the folder 'docs'.

Reply:
I have added a '/' for all.